### ISSUE TITLE
Build breaks with AGP 3.6-beta01

### ISSUE THREAD

**Issue by mateherber** [bugsnag/bugsnag-android-gradle-plugin#177]
### Expected behavior
Build finishes

### Observed behavior
Build halts with error

### Steps to reproduce
Update Android Gradle Plugin to latest 3.6 beta

### Version
4.7.0

### Additional information
output:

```
./gradlew :sonya:processBugsnagDev-debugManifest                                                                                                                                                                                                                     /c/Projects/sonya-android
> Task :sonya:processBugsnagDev-debugManifest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':sonya:processBugsnagDev-debugManifest'.
> Cannot cast object 'property(interface org.gradle.api.file.Directory, fixed(class org.gradle.api.internal.file.DefaultFilePropertyFactory$FixedDirectory, C:\Projects\sonya-android\sonya\build\intermediates\bundle_manifest\devDebug\bundle-manifest))' with class 'org.gradle.api.internal.file.DefaultFilePropertyFactory$DefaultDirectoryVar' to class 'java.io.File'

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

**Comment by mattdyoung** [comment]
Hi @mateherber 

Thanks for the report. We're just looking into the cause.

**Comment by mateherber** [comment]
Hi @mattdyoung 

Thank you, probably relevant google tracker issue: https://issuetracker.google.com/issues/139371052

This looks like some breaking change in the AGP?

**Comment by dmarcato** [comment]
Same issue here - apparently Google changed `ManifestProcessorTask` from
```java
public File getBundleManifestOutputDirectory()
```
to
```java
public abstract DirectoryProperty getBundleManifestOutputDirectory()
```
and it's breaking https://github.com/bugsnag/bugsnag-android-gradle-plugin/blob/2f626c7c2b0e3fd69707a71e5cc51b13adb8470f/src/main/groovy/com/bugsnag/android/gradle/BugsnagPlugin.groovy#L295

Any timeline for a fix?

**Comment by mattdyoung** [comment]
Fixed in [v4.7.1](https://github.com/bugsnag/bugsnag-android-gradle-plugin/releases/tag/v4.7.1)
